app/views/grades/_orca_output.html.erb

```diff
@@ -19,7 +24,7 @@
       <div style="background-color: #eaeaea; padding: 15px 0 10px 10px; cursor: pointer;"
         class="collapsed" data-toggle="collapse" data-target="#detailed-build"
         id="details-collapse">
-        <h3 style="margin: 0 0 0 2px;">Detailed Build</h3>
+        <h3 style="margin: 0 0 0 2px;">Detailed Build <small><span id="details-chevron" class="glyphicon glyphicon-chevron-right"></span></small></h3>
```

Comment:
Nowhere else in the UI do we use `glyphicon-chevron-right`.  Use `glyphicon-chevron-up` and `-down`, and mimic the JS that toggles the expansion.